### DISCUSSION TITLE
The generated hello-world-korge temptate is not able to build

### DISCUSSION THREAD

**Discussion by AShynkevich** [korlibs/korge#2280] [Q&A]
Once I've run the Korge IDE, I tried to create simple hello-world-project:
<img width="784" alt="image" src="https://github.com/user-attachments/assets/4238b833-b8d8-449b-b82c-7c10da82f9db">

After the project initialized/indexed all necessary stuff I disabled the targets, and left only two: targetJvm() & targetAndroid()
<img width="501" alt="image" src="https://github.com/user-attachments/assets/65a3cf77-3586-4013-be10-4f8bc246509d">

Then I try to run the project manually via main.kt
<img width="781" alt="image" src="https://github.com/user-attachments/assets/c4bcd946-2f9e-4973-bc1e-55adaea6a68a">

and get
<img width="890" alt="image" src="https://github.com/user-attachments/assets/59071093-51f3-4ded-b5b6-b14f8630518b">

Then I try to build via gradle:
`./gradlew clean build`

But get: 
```
Configuration cache state could not be cached: field `__compileLintModelMetadataFileCollection__` of `com.android.build.gradle.internal.lint.JavaArtifactInput` bean found in field `__testArtifact__` of `com.android.build.gradle.internal.lint.VariantInputs` bean found in field `__variantInputs__` of task `:deps:generateReleaseLintModel` of type `com.android.build.gradle.internal.lint.LintModelWriterTask`: error writing value of type 'org.gradle.api.internal.file.collections.DefaultConfigurableFileCollection'
> Could not resolve all files for configuration ':deps:releaseUnitTestCompileClasspath'.
   > Could not resolve org.jetbrains.kotlin:kotlin-test:2.0.0.
     Required by:
         project :deps
      > Unable to find a variant of org.jetbrains.kotlin:kotlin-test:2.0.0 providing the requested capability org.jetbrains.kotlin:kotlin-test-framework-junit:
           - Variant compile provides org.jetbrains.kotlin:kotlin-test:2.0.0
           - Variant enforced-platform-compile provides org.jetbrains.kotlin:kotlin-test-derived-enforced-platform:2.0.0
           - Variant enforced-platform-runtime provides org.jetbrains.kotlin:kotlin-test-derived-enforced-platform:2.0.0
           - Variant javadoc provides org.jetbrains.kotlin:kotlin-test:2.0.0
           - Variant platform-compile provides org.jetbrains.kotlin:kotlin-test-derived-platform:2.0.0
           - Variant platform-runtime provides org.jetbrains.kotlin:kotlin-test-derived-platform:2.0.0
           - Variant runtime provides org.jetbrains.kotlin:kotlin-test:2.0.0
           - Variant sources provides org.jetbrains.kotlin:kotlin-test:2.0.0


* Try:
> Review the variant matching algorithm at https://docs.gradle.org/8.8/userguide/variant_attributes.html#sec:abm_algorithm.
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

```

Is it supposed to be or I can fix it somehow via **korge** plugin?

The **hello-world** project I pushed here: https://github.com/AShynkevich/korge-test

**Comment by soywiz** [comment]
Hi @AShynkevich,

For this kind of errors could you crea create an issue here?: https://github.com/korlibs/korge/issues
Q&A is more like for figuring out how to do things that are not errors.

For this problem. Could you try to comment or remove these lines:

https://github.com/AShynkevich/korge-test/blob/master/gradle.properties#L6-L7

Also try to set https://github.com/AShynkevich/korge-test/blob/master/gradle/libs.versions.toml#L2

to `korge = { id = "com.soywiz.korge", version = "6.0.0-beta2" }`

**Comment by AShynkevich** [comment]
Hi @soywiz,

FYI, The result of the actions I published in the issue discussion: https://github.com/korlibs/korge/issues/2283
